frontend/src/assets/stylesheets/components/post-buttons.sass

```diff
@@ -46,10 +49,10 @@
     width: 180px
 
   &__button-logout
-    background: none;
```

Comment:
🤣

Reply:
Weird i do not have semicolon there

![Screenshot 2021-06-17 at 13 49 05](https://user-images.githubusercontent.com/47920348/122390965-d0b9f000-cf72-11eb-98af-aab08c872b0b.png)

Reply:
hmm, is about semicolon?